### ISSUE TITLE
Secondary vesicoureteral reflux

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6822]
**Preferred term label:**
Secondary vesicoureteral reflux

**Synonyms**

**Definition (free text, please give PubMed ID)**
Vesicoureteral reflux induced by increased bladder pressures in patients with voiding dysfunction e.g. in case of congenital posterior urethral valves or neurogenic bladder dysfunction.

PMID: 19668250, 18322164
**Parent term (use hpo.jax.org/app)**
Vesicoureteral reflux HP:0000076

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
